From 7ad98ffa30d1a108654f4a11f5a8f983834821a4 Mon Sep 17 00:00:00 2001
Message-ID: <7ad98ffa30d1a108654f4a11f5a8f983834821a4.1736059920.git.divya@subvertising.org>
In-Reply-To: <cover.1736059918.git.divya@subvertising.org>
References: <cover.1736059918.git.divya@subvertising.org>
From: Divya Ranjan <divya@subvertising.org>
Date: Sat, 4 Jan 2025 21:45:56 +0000
Subject: [PATCH 43/44] gnu: rust-clap-complete-nushell-4: Update to 4.5.5.

* gnu/packages/crates-io.scm (rust-clap-complete-nushell-4): Update to 4.5.5.
---
 gnu/packages/crates-io.scm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/crates-io.scm b/gnu/packages/crates-io.scm
index 5c1a16a826..d34550ba78 100644
--- a/gnu/packages/crates-io.scm
+++ b/gnu/packages/crates-io.scm
@@ -12899,14 +12899,14 @@ (define-public rust-clap-complete-fig-3
 (define-public rust-clap-complete-nushell-4
   (package
     (name "rust-clap-complete-nushell")
-    (version "4.5.4")
+    (version "4.5.5")
     (source
      (origin
        (method url-fetch)
        (uri (crate-uri "clap_complete_nushell" version))
        (file-name (string-append name "-" version ".tar.gz"))
        (sha256
-        (base32 "0xvnl61gnc3j76b9y50y35zvg7fls30i7lyb43fmsvncj3kh4n9i"))))
+        (base32 "12miqxh9g7q37w11bgv55b32s0hdf6avf0lhagzc5psp6icv3a66"))))
     (build-system cargo-build-system)
     (arguments
      `(#:cargo-inputs (("rust-clap" ,rust-clap-4)
-- 
2.47.1

